site/_docs/reference.md

```diff
@@ -2650,6 +2650,7 @@ The 'C' (compatibility) column contains value:
 * 'q' for Microsoft SQL Server ('fun=mssql' in the connect string),
 * 'o' for Oracle ('fun=oracle' in the connect string),
 * 'p' for PostgreSQL ('fun=postgresql' in the connect string),
+* 'r' for Amazon RedShift ('fun=redshift' in the connect string),
```

Comment:
I see that you explicitly say when a function is in redshift. That's good.

Does the test check that the table in reference.md is consistent with the functions defined in SqlLibaryOperators?

Reply:
Yes, this does verify that `reference.md` is consistent with the function libraries (including Redshift). I tested removing `r` from the `CHR` function in `reference.md` and there was a failed test as expected.